docs/sources/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule.md

```diff
@@ -112,6 +112,23 @@ The following template variables are available when expanding annotations and la
 | $values | The values of all reduce and math expressions that were evaluated for this alert rule. For example, `{{ $values.A }}`, `{{ $values.A.Labels }}` and `{{ $values.A.Value }}` where `A` is the `refID` of the expression. This is unavailable when the rule uses a classic condition. |
 | $value  | The value string of the alert instance. For example, `[ var='A' labels={instance=foo} value=10 ]`.                                                                                                                                                                                  |
 
+#### Template functions
+
+The following template functions are available when expanding annotations and labels.
+
+| Names              | Arguments        | Returns                | Descriptions                                                                                                                       |
```

Comment:
```suggestion
| Name             | Argument       | Return                | Description                                                                                                                       |
```

Reply:
Sorry for asking for a change again. The docs squad came to a consensus to use singular.

Reply:
No problem! Are you sure you want it to say "Return" instead of "Returns"?